### PR TITLE
Enhance helmChart structure

### PR DESCRIPTION
**Pull request by xiangfu0**
- Adding comments for docker image tags
- Make a hyper link of helmChart from root directory: https://github.com/apache/pinot/issues/11647

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1715,6 +1715,7 @@
             <!-- Docker and Kubernetes (not part of the distribution) -->
             <exclude>kubernetes/**</exclude>
             <exclude>docker/**</exclude>
+            <exclude>helm/**</exclude>
```
Comment (walterddr) [inline pom.xml:1718]: why do we need this?

**Review by walterddr — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (xiangfu0) [reply pom.xml:1718]: I created a hyper link of helm from root

Comment (xiangfu0) [reply pom.xml:1718]: Ref: https://github.com/apache/pinot/issues/11647

Comment (walterddr) [reply pom.xml:1718]: ah. OK

**Pull request merged**